src/app/models/propTypes/index.js

```diff
@@ -41,27 +41,25 @@ export const imagePropTypes = {
 };
 
 export const videoPropTypes = {
-  model: shape({
-    blocks: arrayOf(
-      shape({
-        locator: string,
-        blocks: arrayOf(
-          // raw video
-          shape({
-            model: shape({
-              isLive: bool,
-              duration: string,
-              locator: string.isRequired,
-            }),
+  blocks: arrayOf(
+    shape({
+      locator: string,
```

Comment:
I think the locator is the same as the locator in the raw video block so technically should be required as well

Reply:
Possibly, but that is outside the remit of this PR really. Intend to open more issues to review props of Image and Video, and move them to the new format.